src/sequence-steps.js

```diff
@@ -279,7 +248,10 @@ export function gitPushOriginMaster( [ git, options ] ) {
 export function githubUpstream( [ git, options ] ) {
 	const command = `git config remote.upstream.url`;
 	return utils.exec( command ).then( data => {
-		const [ , owner, name ] = data.match( /github.com[:/](.*)\/(.*)\.git/ ) || [];
+		let [ , owner, name ] = data.match( /github.com[:/](.*)\/(.*)/ ) || []; // eslint-disable-line prefer-const
```

Comment:
If someone's remotes are using https rather ssh, but yr right Regex does that for us! Changing....